src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -144,8 +147,17 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
     val promise = Promise[Seq[Try[Unit]]]()
 
     Future.sequence(batchStatements).onComplete {
-      case Success(_) => promise.complete(Success(Nil)) // Nil == all good
-      case Failure(e) => promise.failure(e)
+      case Success(_) =>
+        for (
+          p <- pubsub;
+          tag: String <- serialized.map(_.payload.map(_.tags).flatten).flatten.toSet
+        ) {
+          p ! DistributedPubSubMediator.Publish("akka.persistence.cassandra.journal.tag", tag)
```

Comment:
Agreed. I think a simple "wait Xms before publishing next msg" will do. Would 10ms be a sensible default?